lib/checkMergeConflicts.js

```diff
@@ -1,54 +1,85 @@
-module.exports.checkMergeConflicts = async function(context) {
-  var mergeConflictLabel = ['PR: don\'t merge - HAS MERGE CONFLICTS'];
-  pullRequestsPromiseObj = await context.github.pullRequests.getAll(
-    context.repo({per_page: 40}));
+const mergeConflictLabel = ['PR: don\'t merge - HAS MERGE CONFLICTS'];
```

Comment:
Instead of referring to mergeConflictLabel[0] everywhere, why don't we just declare this as a string?

Reply:
Done, thanks!